modules/rosa-list-objects.adoc

```diff
@@ -0,0 +1,443 @@
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster for which the cluster administrators will be listed.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Example
+List all of the cluster administrators and dedicated administrators for a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa list users --cluster=mycluster
+----
+
+[id="rosa-list-versions_{context}"]
+== list versions
+
+Lists all of the OpenShift versions that are available for creating a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa list versions [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|--channel-group
+|Lists only versions from the specified channel group (string). Default: stable
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Example
+List all OpenShift versions.
+
+[source,terminal]
+----
+$ rosa list versions
+----
+
+[id="rosa-describe-admin_{context}"]
+== describe admin
+
+Show the details of a specified cluster-admin user and a command to log in to the cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa describe admin [ID | Name] [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster to which the cluster-admin belongs.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Example
+Describe the cluster-admin user for a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa describe admin --cluster=mycluster
+----
+
+[id="rosa-describe-cluster_{context}"]
+== describe cluster
+
+Shows the details for a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa describe cluster [ID | Name] [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+.Examples
+Describe a cluster named 'mycluster'.
+[source,terminal]
+----
+$ rosa describe cluster mycluster
+----
+
+Describe a cluster using the `--cluster` argument.
```

Comment:
As mentioned earlier, it's not necessary to show both options of how to do the same thing. It confused me in an earlier section thinking that they were doing something different. Just one (a recommended way) is all we'd want to show.

Reply:
Done.